app/static/js/app.js

```diff
@@ -247,6 +245,53 @@ function sendPastedText(pastedText) {
   document.getElementById("app").focus();
 }
 
+function restoreCursor() {
+  if (
+    "settings" in window &&
+    "cursor" in window.settings &&
+    window.settings.cursor != null
+  ) {
+    setCursor(window.settings.cursor);
+  } else {
+    setCursor(3);
+  }
+}
+
+function setCursor(e, save = true) {
+  let i = 0,
+    cursorlist = document.getElementById("cursor-list").firstChild;
+  while (save && cursorlist) {
+    if (cursorlist.nodeType === 1) {
+      i++;
+      cursorlist.firstChild.removeAttribute("class");
+      if (i === e) {
+        cursorlist.firstChild.classList.add("nav-selected");
+      }
+    }
+    cursorlist = cursorlist.nextSibling;
+  }
+  let cursors = [
+    "disabled",
+    "default",
+    "none",
+    "crosshair",
+    "dot",
+    "pointer",
+    "cell",
+  ];
+  if (cursors.length > e) {
+    const el = document.getElementById("remote-screen-img");
+    if (save) {
+      window.settings.cursor = e;
+    }
+    if (connectedToServer) {
+      el.removeAttribute("class");
+      el.classList.add("cursor-" + cursors[e]);
+    }
+  }
+  return false;
```

Comment:
Is this just for the click handler? I'd prefer to do it in the `onclick` because the return value doesn't mean anything here.

Reply:
you are right it is not required, returning nothing is essentially returning false;

return false prevents the anchor url from firing tinypilot/#
(Even though the power button on the top right still takes you to /#)